keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -0,0 +1,441 @@
+
+    return x
+
+
+@keras.utils.register_keras_serializable(package="keras_cv.models")
+class MobileNetV3Backbone(Backbone):
+    """Instantiates the MobileNetV3 architecture.
+
+    References:
+        - [Searching for MobileNetV3](https://arxiv.org/pdf/1905.02244.pdf)
+        (ICCV 2019)
+        - [Based on the Original keras.applications MobileNetv3](https://github.com/keras-team/keras/blob/master/keras/applications/mobilenet_v3.py)
+
+    For transfer learning use cases, make sure to read the
+    [guide to transfer learning & fine-tuning](https://keras.io/guides/transfer_learning/).
+
+    Args:
+        stackwise_expansion: list of ints or floats, the expansion ratio for
+            each inverted residual block in the model.
+        stackwise_filters: list of ints, number of filters for each inverted
+            residual block in the model.
+        stackwise_stride: list of ints, stride length for each inverted
+            residual block in the model.
+        filters: integer, the number of filters for the convolution layer.
+        include_rescaling: bool, whether to rescale the inputs. If set to True,
+            inputs will be passed through a `Rescaling(scale=1 / 255)`
+            layer.
+        input_shape: optional shape tuple, defaults to (None, None, 3).
+        input_tensor: optional Keras tensor (i.e., output of `layers.Input()`)
+            to use as image input for the model.
+        alpha: float, controls the width of the network. This is known as the
+            depth multiplier in the MobileNetV3 paper, but the name is kept for
+            consistency with MobileNetV1 in Keras.
+            - If `alpha` < 1.0, proportionally decreases the number
+                of filters in each layer.
+            - If `alpha` > 1.0, proportionally increases the number
+                of filters in each layer.
+            - If `alpha` = 1, default number of filters from the paper
+                are used at each layer.
+        dropout_rate: a float between 0 and 1 denoting the fraction of input
+            units to drop, defaults to 0.2.
+
+    Examples:
+    ```python
+    input_data = tf.ones(shape=(8, 224, 224, 3))
+
+    # Randomly initialized backbone with a custom config
+    model = MobileNetV3Backbone(
+        stackwise_expansion=[1, 72.0 / 16, 88.0 / 24, 4, 6, 6, 3, 3, 6, 6, 6],
+        stackwise_filters=[16, 24, 24, 40, 40, 40, 48, 48, 96, 96, 96],
+        stackwise_stride=[2, 2, 1, 2, 1, 1, 1, 1, 2, 1, 1],
+        filters=1024,
+        include_rescaling=False,
+    )
+    output = model(input_data)
+    ```
+    """  # noqa: E501
+
+    def __init__(
+        self,
+        *,
+        stackwise_expansion,
+        stackwise_filters,
+        stackwise_stride,
+        filters,
+        include_rescaling,
+        input_shape=(None, None, 3),
+        input_tensor=None,
+        alpha=1.0,
+        dropout_rate=0.2,
+        **kwargs,
+    ):
+        activation = layers.ReLU()
```

Comment:
It's weird to make `activation` a param in the private methods and then hardcode it here. Either we should expose it as a top-level param or hardcode it everywhere. What do you think @ianstenbit?

Reply:
Perhaps hardcoding it everywhere would be better.